cpp/bench/ann/src/common/ann_types.hpp

```diff
@@ -24,6 +24,11 @@
 
 namespace raft::bench::ann {
 
+enum Objective {
+  THROUGHPUT,  // See how many vectors we can push through
+  LATENCY      // See how many
```

Comment:
maybe add some text here? 
```suggestion
  LATENCY      // See how fast we can push a vector through
```